### DISCUSSION TITLE
How to remove an empty music provider

### DISCUSSION THREAD

**Discussion by papabak** [music-assistant/support#1933] [Q&A] [answered]
Previously I tried to connect the music collection on my NAS, but I did it the wrong way. Of course that didn't work. This has been solved.
However, I would like to remove the now empty provider. It is not really a problem as it doesn't do anything harmful. It more that I wonder if the menu lacks the remove option ... or that I simply haven't found the option.

It is the first line in the screen shot showing "URL".

![image](https://github.com/music-assistant/hass-music-assistant/assets/97854063/5310d1e2-8fc5-4b56-b4ec-f2efbdf691b5)

**Comment by OzGav** [comment, marked as answer]
You can’t remove that one as it is a functionality of MA

**Comment by papabak** [comment]
Ok, then I’ll leave it there. Just wanted to know if I missed an option to remove it. Thanks!
Dirk
________________________________
Van: OzGav ***@***.***>
Verzonden: Tuesday, February 20, 2024 11:28:11 PM
Aan: music-assistant/hass-music-assistant ***@***.***>
CC: papabak ***@***.***>; Author ***@***.***>
Onderwerp: Re: [music-assistant/hass-music-assistant] How to remove an empty music provider (Discussion #1933)


You can’t remove that one as it is a functionality of MA
